config/space/provider_kcp.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: space.kubestellar.io/v1alpha1
+kind: SpaceProviderDesc
+metadata:
+  name: default
+spec:
+  ProviderType: "kcp"
+  SpacePrefixForDiscovery: "espw"
+  Config: |
```

Comment:
Because

```go
	Config string `json:"Config,omitempty"`
```

specifying the empty string as the value for this field is the same as not mentioning this field.

At least, I am sure that would be true if it had been marked `// +optional` (as it should); I think the `json` tag rules in this matter.

Reply:
BTW, the convention in the design of Kubernetes API objects is to use lowerCamelCase for field names in JSON.

Reply:
Agree, will be fixed in dedicated PR.